scriptmodules/ports/hexen2.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="hexen2"
+rp_module_desc="Hexen II - Hammer of Thyrion source port"
+rp_module_licence="GLP2"
```

Comment:
GPL2

Reply:
It should contain a link to the licence text on github

Reply:
What if the git source does not have a copying or licensing.md file applied?

Reply:
Then it can be omitted but most projects do. This repo has a licence in the docs folder.

Reply:
Thank you. I have found the license(copying.txt) file in the docs and it has been included in the latest commits.